nevergrad/functions/images/imagelosses.py

```diff
@@ -45,3 +86,12 @@ def __init__(self, reference: tp.Optional[np.ndarray] = None) -> None:
     def __call__(self, img: np.ndarray) -> float:
         loss = self.koncept.assess(img)
```

Comment:
Koncept 512 takes images of dimension (x, y, 3) and containing float between 0 and 255

Reply:
R, G, B ?